readme.md

```diff
@@ -217,7 +217,7 @@ This folder contains application development guides and device setup instruction
 
 Contains Azure IoT Hub client components that provide the raw messaging capabilities of the library. Refer to the API documentation and samples for information on how to use it.
 
-   * build: has one subfolder for each platform (e.g. Windows, Linux, Mbed). Contains makefiles, batch files, and solutions that are used to generate the library binaries.
```

Comment:
Nits!  Some additional floating "mbed" references.

There are more mbed build directories

```
iothub_client\build\iothub_amqp_transport\mbed
iothub_client\build\iothub_http_transport\mbed
iothub_client\build\iothub_mqtt_transport\mbed
iothub_client_sample_upload_to_blob\mbed
iothub_client\samples\iothub_client_sample_upload_to_blob_mb\mbed
iothub_client\samples\iothub_ll_c2d_sample\mbed
samples\iothub_ll_client_x509_sample\mbed
samples\iothub_ll_telemetry_sample\mbed
serializer\build\mbed
doc\media\mbed1.* # This seems to be not linked by any docs but you should double check
```

* In `serializer\readme.md` please remove the "mbed" reference.

```
you will find detailed instructions on how to use on Linux, mbed, Windows and other platforms
```

In `iothub_client\samples\iothub_client_device_twin_and_methods_sample\iothub_client_device_twin_and_methods_sample.c` 

```c
// For mbed add the certificate information
```

Recomend replacing this with our telemetry sample notes on trusted certificates
```c
        // Setting the Trusted Certificate. This is only necessary on systems without
        // built in certificate stores.
```

We have `MBED_BUILD_TIMESTAMP` #ifdef's floating around in samples and long haul tests I'm assuming are MBED OS and not MBED TLS related but you should double check me.

Reply:
I am really surprised I missed those. Wow. Thanks for the list and sorry for missing them.

MBED_BUILD_TIMESTAMP is related to MBED OS, not mbed TLS, and should be removed as well. 
https://github.com/ARMmbed/mbed-os/issues/10228

Reply:
Also note around line 70 another stray mbed reference - I think you can safely remove the ",mbed".
```
Socket                    | [socketio\_\*](https://github.com/Azure/azure-c-shared-utility/tree/master/adapters)                                | Provides a wrapper over the specific socket API ([win32, berkeley, mbed](https://github.com/Azure/azure-c-shared-utility/tree/master/adapters)), using the [xio](https://github.com/Azure/azure-c-shared-utility/blob/master/inc/azure_c_shared_utility/xio.h) interface that the device client SDK uses
```

Reply:
Fixed. Thanks!